frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -176,9 +176,7 @@ export const surveyLogic = kea<surveyLogicType>([
         },
         installSurveyPlugin: async (_, breakpoint) => {
             actions.setInstallingPlugin(true)
-            const installPl = async (): Promise<void> =>
-                actions.installPlugin('https://github.com/PostHog/feature-surveys', PluginInstallationType.Repository)
-            await installPl()
```

Comment:
FYI @liyiy this doesn't work. Actions don't return anything, it's the listeners that might use promises. Wrapping it in an async caller is doesn't do anything. Happy to explain some of the Kea stuff further if there are some misunderstandings here 👍